src/main/java/org/jabref/logic/importer/fetcher/IacrEprintFetcher.java

```diff
@@ -37,6 +38,7 @@
     private static final Predicate<String> IDENTIFIER_PREDICATE = Pattern.compile("\\d{4}/\\d{3,5}").asPredicate();
     private static final String CITATION_URL_PREFIX = "https://eprint.iacr.org/eprint-bin/cite.pl?entry=";
     private static final String DESCRIPTION_URL_PREFIX = "https://eprint.iacr.org/";
+    private static final Charset WEBSITE_CHARSET = Charset.forName("iso-8859-1");
```

Comment:
very very minor, you can directly use the predefined enum Constant: 
https://docs.oracle.com/javase/8/docs/api/java/nio/charset/StandardCharsets.html#ISO_8859_1

Reply:
Thanks for pointing that out - was looking for something like that, but apparently didn't look long enough...

Reply:
For other cases where Java provides default enum variables, they all start with StandardXXX, for example for file opening there exists: StandardOpenOption, not really obvious if you search for it ;)

Reply:
Jep, I expected them in something like Charsets or directly as constants in the Charset class...